docs/data-ai/ai/deploy.md

```diff
@@ -5,5 +5,23 @@ weight: 40
 layout: "docs"
 type: "docs"
 no_list: true
-description: "TODO"
+description: "Deploy an ML model to your machine."
 ---
+
```

Comment:
I am wondering here if we should merge relevant bits from the ML model service page into this and then get rid of that page in the reference section. It feels somewhat possible? Wondering the same about the vision service and data mgmt service tbh but this seems a good place to start

And maybe also cover https://docs.viam.com/registry/ml-models/#model-framework-support ?